### DISCUSSION TITLE
No backend on brand new install

### DISCUSSION THREAD

**Discussion by gissirob** [wintercms/winter#442] [Q&A] [answered]
I just successfully installed Winter using the web-based installer. Upon completion, I could navigate to the demo front end, but I could not access the backend. (404 Not found). When I checked the filesystem, there was no backend directory off my site root. Help please.

**Comment by gissirob** [comment, marked as answer]
Actually, I answered my own question. It was an Apache configuration issue.

**Reply by bennothommo** [reply]
@gissirob would you be able to describe the problem and solution, for the benefit of other users who might encounter the same problem?

**Reply by gissirob** [reply]
Sure. And I would recommend adding this to the installation docs for those like me with limited Apache knowledge.

The problem was that I could not navigate to the backend, or any of the pages of the demo app. This was because I hadn't added the appropriate rules to the default Apache config file. Basically, I followed the steps in section 1 of the following: https://www.vultr.com/docs/how-to-install-and-configure-winter-cms-on-ubuntu-20-04/. I hope this helps.

**Reply by bennothommo** [reply]
Thanks for that @gissirob - we do cover some Apache configuration in the docs here: https://wintercms.com/docs/setup/configuration#apache-configuration. I think that Vultr doc does cover everything else though. Was there a particular part that tripped you up?

**Reply by gissirob** [reply]
Specifically, it was adding the <Directory /var/www/html/> ... </Directory> section to /etc/apache2/sites-enabled/000-default.conf. I had already done the other steps, but I found the document very thorough. Not sure however, how applicable it is outside of Ubuntu 20.04, which is what I'm using.

**Reply by bennothommo** [reply]
Cool - I'll see if it's something we should include in the docs. I think it'll applicable to any distro that use Apache, so it's probably worth mentioning.

**Comment by hsvtarget** [comment]
For anyone else who followed a similarly crooked path to get here as I did: Changing non-SSL config may not solve your problem if you try to install using SSL and your SSL config file needs the same change outlined above.

In messing about with attempting to install a different, much less friendly CMS, I had duplicated server doc/dir specs in the SSL config for Apache. Mirroring the additions to the SSL config resolved the issue, everything now works a treat!
